app/code/Magento/Sales/Test/Mftf/ActionGroup/AdminSetCustomPriceActionGroup.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AdminSetCustomPriceActionGroup">
```

Comment:
I would suggest providing more detailed naming for the action groups. So, the name of the action group describes its purpose in detail. For example, the action group above `AdminFillAccountInformationOnCreateOrderPageActionGroup` ideally describes the case. So we know where exactly we fill the account information. With `AdminSetCustomPriceActionGroup` it's not obvious where it's reusable. 

Hope it makes sense.

Thanks

Reply:
@rogyar naming has been adjusted, thank you!